addons/event_system_plugin/plugin_script.gd

```diff
@@ -76,7 +76,7 @@ func make_visible(visible: bool) -> void:
 
 func save_external_data() -> void:
 	if is_instance_valid(_timeline_editor):
-		_timeline_editor.call_deferred("_update_values")
+		_timeline_editor.call("_update_values")
```

Comment:
Notice that this method does not saves anything, just updates the timeline editor interface when the editor saves everything